modules/core/test/src/smithy4s/http/MatchPathSpec.scala

```diff
@@ -16,16 +16,66 @@
 
 package smithy4s.http
 
-import weaver.FunSuite
+import cats.Show
+import cats.data.NonEmptyList
+import org.scalacheck.Arbitrary
+import org.scalacheck.Gen
+import weaver.SimpleMutableIOSuite
+import weaver.scalacheck.Checkers
+import smithy4s.http.PathSegment.GreedySegment
+import smithy4s.http.PathSegment.LabelSegment
+import smithy4s.http.PathSegment.StaticSegment
 
-object MatchPathSpec extends FunSuite {
+object MatchPathSpec extends SimpleMutableIOSuite with Checkers {
```

Comment:
Small nitpick but `SimpleMutableIOSuite` is aliased as `SimpleIOSuite` in weaver.

Reply:
right, Metals doesn't help with alias suggestions 😅